spec/trips_performed.schema.json

```diff
@@ -0,0 +1,187 @@
+{
+  "primaryKey": [
+    "date",
+    "trip_id_performed"
+  ],
+  "missingValues": [
+    "NaN",
+    ""
+  ],
+  "fields": [
+    {
+      "name": "date",
+      "type": "date",
+      "description": "Service date. References GTFS",
+      "constraints": {
+        "required": true
+      }
+    },
+    {
+      "name": "trip_id_performed",
+      "type": "string",
+      "description": "Identifies the trip performed.",
+      "constraints": {
+        "required": true
+      }
+    },
+    {
+      "name": "vehicle_id",
+      "type": "string",
+      "title": "ID referencing vehicles.vehicle_id",
+      "description": "Identifies the vehicle.",
+      "constraints": {
+        "required": true
+      }
+    },
+    {
+      "name": "trip_id_scheduled",
+      "type": "string",
+      "title": "ID referencing Trips.trip_id in GTFS",
+      "description": "Identifies the scheduled trip associated with the trip performed. One scheduled trip may be associated with multiple operated trips, or an operated trip may not be associated with a scheduled trip. References GTFS"
+    },
+    {
+      "name": "route_id",
+      "type": "string",
+      "title": "ID referencing routes.route_id",
+      "description": "Identifies the route. References GTFS"
+    },
+    {
+      "name": "route_type",
+      "type": "integer",
+      "description": "Indicates the type of transportation used on a route. Valid options are:\n0 - Tram, Streetcar, Light rail. Any light rail or street level system within a metropolitan area.\n1 - Subway, Metro. Any underground rail system within a metropolitan area.\n2 - Rail. Used for intercity or long-distance travel.\n3 - Bus. Used for short- and long-distance bus routes.\n4 - Ferry. Used for short- and long-distance boat service.\n5 - Cable tram. Used for street-level rail cars where the cable runs beneath the vehicle, e.g., cable car in San Francisco.\n6 - Aerial lift, suspended cable car (e.g., gondola lift, aerial tramway). Cable transport where cabins, cars, gondolas, or open chairs are suspended by means of one or more cables.\n7 - Funicular. Any rail system designed for steep inclines.\n11 - Trolleybus. Electric buses that draw power from overhead wires using poles.\n12 - Monorail. Railway in which the track consists of a single rail or a beam. References GTFS",
+      "constraints": {
+        "enum": [
+          0,
+          1,
+          2,
+          3,
+          4,
+          5,
+          6,
+          7,
+          8,
+          9,
+          10,
+          11,
+          12
+        ]
+      }
+    },
+    {
+      "name": "pattern_id",
+      "type": "string",
+      "title": "ID referencing proposed trips.pattern_id in GTFS",
+      "description": "Identifies the exact service and path on the route. References GTFS"
```

Comment:
> There is no "pattern_id" in the GTFS standard

I wish there was though :-)
Wondering if this even can be `shape_id` since it isn't a required GTFS field? it certainly makes the most sense but 🤔

Reply:
Removing `pattern_id` entirely.